### ISSUE TITLE
upload photo"gif"

### ISSUE THREAD

**Issue by lunafreya4** [GeekMagicClock/HelloCubic-Lite#8]
why it keep saying error 404 ?
i just resized gif to 270 and click on upload

**Comment by GeekMagicClock** [comment]
Hi, what is your web broswer, please try Chrome instead, and do not use translation on the web.

By the way, the full screen pixel is 240, not 270px

**Comment by lunafreya4** [comment]
> Hi, what is your web browser, please try Chrome instead, and do not use translation on the web.

yes i use google chrome  , thank you i turn off google translate and it works
